### PR TITLE
added tests and template for exercise gif_is_life

### PR DESCRIPTION
**Pull request by AGuyNextDoor**

### CODE REVIEW

**Comment by AGuyNextDoor** [comment]
https://github.com/fewlinesco/sparta_courses/pull/326

**Review by Fenntasy — COMMENTED** [review]
Do you need help for the commented tests? Here you only test the presense of an `input` and a `button`?

File: 06_react_redux/02_theory/04_gif_is_life/src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  expect(searchMock.find('button')).toBeDefined();
+})
+
+test("keyword filled in form gives the right gif's URL from GIPHY", async () => {
+
+  const searchedGif = await Wapiti().goto("http://localhost:3000")
+    .fillForm({"#gifSearcher": "lol"}, { waitForPageLoad: false })
+    .puppeteer(page => page.waitForSelector("img"))
+    // .
```
Comment (Fenntasy) [inline 06_react_redux/02_theory/04_gif_is_life/src/App.test.js]: You can remove this dot 😛

**Review by Fenntasy — COMMENTED** [review]
Just remove the useless comment and it will be good for me

**Review by Fenntasy — APPROVED** [review]

**Pull request merged**